test/common/swaggerSpec.js

```diff
@@ -14,13 +14,9 @@
 'use strict';
 
 var chai = require('chai');
```

Comment:
since we had a discussion about not making chai as global variable, I have not made any changes for this, this particular file using chai to add some custom logic middleware, so I think we can keep this one as it is.